cmd/root.go

```diff
@@ -609,6 +610,13 @@ func newRunCommand(ctx context.Context, input *Input) func(*cobra.Command, []str
 
 		cancel := artifacts.Serve(ctx, input.artifactServerPath, input.artifactServerAddr, input.artifactServerPort)
 
+		// TODO: provide a way to configure the cache handler via cli
+		cacheHandler, err := artifactcache.StartHandler("", "", 0, common.Logger(ctx))
+		if err != nil {
+			return err
+		}
+		envs["ACTIONS_CACHE_URL"] = cacheHandler.ExternalURL() + "/"
```

Comment:
Can we make this optional? We have an external cache which can run across a cluster of nodes and uses it's own external backend.
Having this here will probably interfer with it, right?

I think the cache handling should be opt-in similar to the asset-server.

Reply:
I think it's fine, as we are setting all required env vars.